e2e/BinaryCoStream/src/DownloaderPeer.tsx

```diff
@@ -1,17 +1,14 @@
 import { Account, BinaryCoStream, ID } from "jazz-tools";
 import { useEffect } from "react";
 import { useAccount, useCoState } from "./jazz";
-import { waitForCoValue } from "./lib/waitForCoValue";
 import { UploadedFile } from "./schema";
 
 async function getUploadedFile(me: Account, uploadedFileId: ID<UploadedFile>) {
-  const uploadedFile = await waitForCoValue(
-    UploadedFile,
-    uploadedFileId,
-    me,
-    Boolean,
-    {},
-  );
+  const uploadedFile = await UploadedFile.load(uploadedFileId, me, {});
+
+  if (!uploadedFile) {
+    throw new Error("Uploaded file not found");
+  }
```

Comment:
When testing locally with the "offline then resume" test I'm getting some consistent failures.

It doesn't happen on CI but need to investigate more nonethless.

Reply:
Found the issue and fixed: https://github.com/gardencmp/jazz/pull/759/commits/bb126009c89c0f332f497e8393cff7f7764afcb3